guides/common/modules/ref_supported-puppet-versions-and-system-requirements.adoc

```diff
@@ -4,9 +4,9 @@
 Before you begin with the Puppet integration, review the supported Puppet versions and system requirements.
 
 Supported Puppet Versions::
-{Project} supports Puppet 7.
-Ensure that the Puppet modules used to configure your hosts are compatible with Puppet 7.
+{Project} supports Puppet 8 and 7.
+Ensure that the Puppet modules used to configure your hosts are compatible with your Puppet version.
```

Comment:
I'm wondering if we should be more specific here regarding the components.

{Project} comes with Puppet server 8. Satellite provides Puppet agent 7 and 8. Both agent versions _should_ work with the Puppet server 8.

Reply:
Added a second sentence. What do you think?